.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,25 @@
+name: Integration Tests
```

Comment:
Forgot to ask - why GitHub Actions over using Drone? 
We may have more powerful machines in Drone. Also would be good to have decent statistics about runtime and success rates. Not sure which option does it better.